app/helpers/application_helper.rb

```diff
@@ -20,8 +20,14 @@ def annict_image_tag(record, field, options = {})
     image_tag(url, options)
   end
 
-  def custom_time_ago_in_words(date)
-    "#{time_ago_in_words(date)}#{t('words.ago')}"
+  def custom_time_ago_in_words(datetime)
+    days = (Time.now.to_date - datetime.to_date).to_i
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.